app/src/marketplace/modules/communityProduct/services.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+
+import { deploy } from '$mp/utils/smartContract'
+import getConfig from '$shared/web3/config'
+
+import type { SmartContractDeployTransaction } from '$shared/flowtype/web3-types'
+import type {
+    StreamId,
+    Stream,
+    NewStream,
+} from '$shared/flowtype/stream-types'
+import type { Permission } from '$userpages/flowtype/permission-types'
+import type { ApiResult } from '$shared/flowtype/common-types'
+
+import { post } from '$shared/utils/api'
+import { formatApiUrl } from '$shared/utils/url'
+import { postStream } from '$userpages/modules/userPageStreams/services'
+
+export const getStreamrEngineAddresses = (): Array<string> => {
```

Comment:
I wonder if this could be done inside webpack (does it make sense)?

Reply:
I think it could be done in webpack. I think this version is more explicit and not something that happens behind the curtains :)

Reply:
That's true :) Anyway, it's not a big deal at this point and easy to refactor later.